doc/Overview/Mvux/WIP_Overview.md

```diff
@@ -0,0 +1,179 @@
+
+Working with immutable data structures can be a challenging transition for developers who have only previously only worked with `INotifyPropertyChanged` implementing classes. The fundamental difference is that since a record is immutable, making a change to it means creating a new object with different data instead of modifying data within an existing one. Don't worry, the "X" part of the MVUX implementation takes care of this for you.
+
+### Extended
+
+The **eXtended** part of the pattern provides the toolset to adapt between the data-binding engine and immutable data objects, as well as the rest of the functionality that will be covered later on.
+
+One of the big differences between MVUX and patterns like MVVM is the use of 'Feeds' in place of some of the change-notification properties. While a property has a single value, a feed is similar to a stream of values.  
+When an updated value is received, it replaces the old value, and the View reacts to the change. In this way, it is similar to reactive programming. However, MVUX additionally combines the Feed's value with 'statuses'.  
+Common statuses for a feed include 'loading', 'has value', 'refreshing', 'empty', and 'error'. MVUX includes controls that make displaying these different statuses in the View easy without the need to create additional properties to manipulate and control what to show.
+
+A feed can be a stream of a single value or a collection of values. It can also keep track of its current value(s). Keeping track of values is necessary if they may need to be 'updated' while providing performance optimizations if you only need to display the value in the View without changing it.
+Models are always created asynchronously and automatically handle any necessary transitions to the UI thread.  
+
+In many ways, MVUX is closer to functional programming than the imperative approach often taken by developers using the MVVM pattern. Writing code this way brings benefits in code reuse and can simplify testing.
+
+The MVUX pattern brings four key benefits over other design patterns used to build native applications.
+
+- It's entirely `async` to ensure a responsive user interface.
+- It is reactive rather than event-driven and so needs less boilerplate code.
+- It encourages the immutability of Model classes, which in turn can lead to improved efficiency and performance, simpler code, and easier object comparison and testing.
+- It automatically tracks and reports the status (or 'state') of asynchronously requested data, which also simplifies the code you need to write. 
+
+Now that you know the fundamentals of the pattern, let's see how to use it to create the simple weather app mentioned above. 
+
+## Using MVUX to create an app
+
+This example creates an `IFeed<WeatherInfo>` that will return the weather, which includes the temperature, loaded asynchronously from a weather service.
```

Comment:
Should focus on the user reading the code and understand it on the page or encourage them to create on? This would include detailing all the steps in creating the project using VS etc. etc.